sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-files/e-cert-file-header.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from "@nestjs/common";
+import { FixedFormatFileLine } from "../../../services/ssh/sftp-integration-base.models";
+import { RecordTypeCodes } from "../models/e-cert-integration-model";
+
+/**
+ * Header of an E-Cert file.
+ * The documentation about it is available on the document
+ * 'CSLP-AppendixF2AsReviewed2016-FileLayouts BC Files V3(HAJ-CB EDITS) In ESDC Folder'.
+ */
+@Injectable()
+export abstract class ECertFileHeader implements FixedFormatFileLine {
+  recordTypeCode: RecordTypeCodes;
```

Comment:
It seems that the base class declares 1 common property that is also declared in all derived classes. Can it be declared only on the base class assuming that they will be mandatories?
Also, there are other 2 variables declared on both derived classes, should we move them to the base?